tests/features/steps/check-ui-steps.py

```diff
@@ -0,0 +1,30 @@
+from behave import *
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service as ChromeService
+from selenium.webdriver.common.by import By
+
+
+@given(u'the user is on the todo list page')
+def open_browser(context):
+    context.driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
+    context.driver.implicitly_wait(5)
+    context.driver.get("http://127.0.0.1:5000/")
+
+
+@then(u'the page should have a text field to enter the title of the task')
```

Comment:
also same as above for unicode strings, this goes for all `u'..'` strings. its just a nitpick tho